content/assets/css/application.scss

```diff
@@ -180,11 +168,6 @@ dfe-icon-container-header {
   margin-left: 8px;
 }
 
-.govuk-list,
```

Comment:
This was to make the markdown page not full width. I added a two-thirds column to the markdown layout instead.